woocommerce/woocommerce-gateway-stripe
# chore: add P24 & Sofort to UPE

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1918
![Screen Shot 2021-09-21 at 3 20 31 PM](https://user-images.githubusercontent.com/273592/134242140-8a907beb-77bc-445e-a6c3-8ed6c521d9dc.png)
![Screen Shot 2021-09-21 at 3 20 04 PM](https://user-images.githubusercontent.com/273592/134242142-424db9b5-8b11-4b1f-9356-0eef9eaa35d5.png)



# Testing instructions
- Ensure you have the `_wcstripe_feature_upe` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- Payment methods have been added
